src/runtime/pythonengine.cs

```diff
@@ -744,7 +698,7 @@ public static KeywordArguments kw(params object[] kv)
 
         public static PyObject Import(string name)
```

Comment:
I think this should also be obsoleted. Having a clear separation of concerns in our API will help us documenting it and keeping it stable for a long time.